packages/html-reporter/src/chip.tsx

```diff
@@ -40,8 +40,9 @@ export const Chip: React.FC<{
       className={clsx('chip-header', setExpanded && ' expanded-' + expanded)}
       onClick={() => setExpanded?.(!expanded)}
       title={typeof header === 'string' ? header : undefined}>
-      {setExpanded && !!expanded && icons.downArrow()}
-      {setExpanded && !expanded && icons.rightArrow()}
+      {setExpanded && !!expanded && <icons.downArrow />}
+      {setExpanded && !expanded && <icons.rightArrow />}
+      {!setExpanded && <icons.spacer />}
```

Comment:
Consider making this a ternary? These all go in the same slot and are mutually exclusive